test/reporter-html/diff.js

```diff
@@ -0,0 +1,163 @@
+QUnit.module( "diff" );
```

Comment:
Why is this in the `reporter-html` folder? The relevant code isn't attached to the reporter.